public/templates/arm-subsystem.java.hbs

```diff
@@ -35,12 +38,10 @@ public class {{subsystemName}} extends SubsystemBase {
  private final double forwardSoftLimit = {{forwardSoftLimit}}; // max angle in radians
  private final double reverseSoftLimit = {{reverseSoftLimit}}; // min angle in radians
  {{/if}}
- {{#if enableStatorLimit}}
+ private final boolean enableStatorLimit = {{enableStatorLimit}};
```

Comment:
This should really be an `{{#if` inside of the template with a true/false. The paradigm of having attributes which for this is kinda bad imo.

Reply:
It feels like non-insignificant duplicated work per motor controller that also takes away configurability from the end user, which for something that serves off as a template feels counter-intuitive

Reply:
I will have to think on this alot then. 

"Do we want code that is a template or generates the cleanest and smallest code possible?"

This decision will drive all future code template changes too 🤔